.circleci/config.yml

```diff
@@ -714,7 +749,7 @@ workflows:
 
       - test-discovery-provider:
           name: test-discovery-provider
-      - docker-build-and-push:
+      - docker-build-and-push-updated:
```

Comment:
can we leave it `build-and-push` bc we're replacing the old one?

Reply:
The updated one uses a new base image - I didn't want to go through the process of updating the build for all the services Base image `circleci/buildpack-deps:stretch` => `cimg/base:2022.03`